src/classes/CW_AudienceCSVProcessHelperTest.cls

```diff
@@ -0,0 +1,145 @@
+/**
+* File Name   :	CW_AudienceCSVProcessHelperTest.cls
+* Description :	tests CW_AudienceCSVProcessHelper and CSVBatchProess
+*
+* Modification Log
+* ============================================================================
+* Ver Date	   Author		 Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex Panka     Initial version
+
+*
+*/
+@isTest
+private class CW_AudienceCSVProcessHelperTest {
+	
+	 public static String csvFile='Group Name,Account Name \nC2,1\nC2,2\nT3,3\nT3,4\nT4,5\nT4,6';
+	
+	
+	static testMethod void UploadBatchOnly()
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility. Style issue: methods should use the "lowerCamelCase" notation. Suggestion: rename accordingly.